src/rust/graph-descriptions/build.rs

```diff
@@ -66,7 +66,7 @@ fn main() {
         "#[builder(default)]",
     );
     config.field_attribute(".graph_description.File.file_name", "#[builder(default)]");
-    config.field_attribute(".graph_description.File.file_path", "#[builder(default)]");
+    // config.field_attribute(".graph_description.File.file_path", "#[builder(default)]");
```

Comment:
should all these comments still be here, or should they be removed?

Reply:
Ahh, yea. I'll probably restore these lines because this didn't work the way I wanted it to but I was hoping to get a compile-time error with derive-builder if certain fields weren't set but I forgot this was a runtime check.